README.md

```diff
@@ -50,6 +50,42 @@ cd sc-machine/scripts
 ./run_sc_server.sh
 ```
 
+For running sc-server in authorization server mode, you need a key pair. Current version of http service supports autogeneration of these keys. However you can done this manually with next commands:
+
+```sh
+openssl genrsa -out private.pem 2048
+openssl rsa -in private.pem -pubout -outform PEM -out public.pem
+```
+
+Current endpoints for auth service:
+
+1. **127.0.0.1:8090/auth/get_tokens** - generate access and refresh token for current user's login and password, params: username, password, role. After check user credentials, service issues access and refresh token for future requests.
+**POST request-body:** {"name": "alex", "password": "a665a45920422f9d417e4867efdc4fb8a04a1f3fff1fa07e998e86f7f7a27ae3"}
+2.  **127.0.0.1:8090/auth/get_access_token** - generate access token, **for this operation needs refresh token**
+**POST request-body:** {"access_token": "..."}
+3. **127.0.0.1:8090/admin/user** - admin endpoint (need specific type of request for correct work), **for this operation needs access token**:
+**POST-request:** add new user,
+**DELETE-request:** delete user by user name,
+**PUT-request:** change user record by user name.
+_**Examples:**_
+**POST request-body:** {"name": "alex123", "password": "a665a45920422f9d417e4867efdc4fb8a04a1f3fff1fa07e998e86f7f7a27ae3", "access_token": "..."}
+**DELETE request-body:** {"name": "alex123", "access_token": "..."}
+**PUT request-body:** {"name": "alex123", "new_name": "alex234", "password": "...", "access_token": "..."}
```

Comment:
PUT method didn't work correctly. Firstly I create new user.
![image](https://user-images.githubusercontent.com/34061914/159484260-2fb58102-986a-45fb-a8af-63cd1840e9cb.png)

After that I try to rename user, but every time response contain message with code number 3. 
![image](https://user-images.githubusercontent.com/34061914/159484380-badde4e4-598f-47c9-941b-051a78f0ff15.png)

Also I use GET method for users, there is my user with first-created name
![image](https://user-images.githubusercontent.com/34061914/159484523-0c0ee02c-45fa-4d83-b45a-e216a3d65524.png)